SimPPS/RPDigiProducer/src/DeadChannelsManager.cc

```diff
@@ -0,0 +1,58 @@
+#include "SimPPS/RPDigiProducer/interface/DeadChannelsManager.h"
+#include "CondFormats/CTPPSReadoutObjects/interface/TotemSymbId.h"
+#include "CondFormats/CTPPSReadoutObjects/interface/TotemDAQMapping.h"
+#include "DataFormats/CTPPSDetId/interface/TotemRPDetId.h"
+#include "SimPPS/RPDigiProducer/interface/RPDisplacementGenerator.h"
+#include <map>
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+DeadChannelsManager::DeadChannelsManager() {
+	analysisMaskPresent = false;
+}
```

Comment:
Actually, why is the default constructor needed? By quick look only the other one is called.